app/services/audio.js

```diff
@@ -61,8 +59,17 @@ export default Service.extend({
 
   init() {
     this.get('hifi').on('audio-ended', () => this.finishedTrack());
+    Ember.$(window).on('beforeunload', () => {
+      if (this.get('currentAudio')) {
+        this.sendListenAction(this.get('currentAudio'), 'close');
+      }
+    });
     this._super(...arguments);
   },
+  
+  willDestroy() {
+    Ember.$(window).off('beforeunload');
```

Comment:
sure yeah. This is really just for testing thought, since the audio service is long-lived and will probably never be destroyed until the window is closed or refreshed.